shared/team-building/user-bubble.js

```diff
@@ -143,10 +166,10 @@ const styles = Styles.styleSheetCreate({
 
   removeIcon: Styles.platformStyles({
     isElectron: {
```

Comment:
not centered in the bubble, `margin: 'auto'` might help
<img width="32" alt="image" src="https://user-images.githubusercontent.com/11968340/53757623-1e85b980-3e8a-11e9-975c-931f02a59572.png">

Reply:
I added that and I think it's ok?